sdk/storage/tests.yml

```diff
@@ -3,30 +3,36 @@ trigger: none
 jobs:
   - template: ../../eng/pipelines/templates/jobs/archetype-sdk-tests.yml
     parameters:
+      Matrix:
+        Windows - Java 8:
+            OSName: 'Windows'
+            OSVmImage: 'windows-2019'
+            JavaVersion: '1.8'
+            DisplayName: 'Windows - Java 8'
       ServiceDirectory: storage
+      TimeoutInMinutes: 240
```

Comment:
Let's use a 2 hour timeout since the complete test suite only takes about 60-75 minutes, anything longer and there is likely an issue in our tests.